Dockerfile

```
@@ -0,0 +1,21 @@
+#Deriving the latest base image
+FROM python:latest
+
+
+#Labels as key value pair
+
+
+
+# Any working directory can be chosen as per choice like '/' or '/home' etc
+# i have chosen /usr/app/src
+WORKDIR /usr/app/src
+
+#to COPY the remote file at working directory in container
+COPY app.py ./
```

Si c'est le ToDoList_CLI.py qu'il faut lancer, ne pas oublier de changer : 
```suggestion
COPY *.py ./
```